linode/manager
# feat: [M3-7016] - Add Basic Filtering for Firewall Devices

## Description 📝
Added device filtering for the Linode and Nodebalancer firewall tabs.

## Major Changes 🔄
- Added device filtering using the `DebouncedSearchTextField` component

## Preview 📷

| Before  | After   |
| ------- | ------- |
| ![Screenshot 2023-09-11 at 6 16 05 PM](https://github.com/linode/manager/assets/139489745/23c4d054-167a-448c-ad21-44563c397029) | ![Screenshot 2023-09-11 at 6 13 56 PM](https://github.com/linode/manager/assets/139489745/4155bf55-0c69-4841-93af-c27c266e90e7) |
| | ![Screenshot 2023-09-11 at 6 15 08 PM](https://github.com/linode/manager/assets/139489745/42b61715-8abe-48cc-877c-0925fa07c991) |

## How to test 🧪
1. Navigate to http://localhost:3000/firewalls
2. Click on any of the firewall
4. Navigate to the linodes tab
5. Make sure to add linodes to the firewall
6. Ensure the filter works properly
7. Navigate to the nodebalancers tab
8. Make sure to add nodebalancers to the firewall
9. Ensure the filter works properly
